rllib/algorithms/bc/tf/bc_tf_rl_module.py

```diff
@@ -0,0 +1,6 @@
+from ray.rllib.algorithms.bc.bc_rl_module import BCRLModule
+from ray.rllib.core.rl_module.tf.tf_rl_module import TfRLModule
+
+
+class BCTfRLModule(TfRLModule, BCRLModule):
+    framework: str = "tf2"
```

Comment:
Dumb question: Should we move the definition `framework: str = "tf2"` (same for torch) into TfRLModule instead? And then just do:
```
class BCTfRLModule(TfRLModule, BCRLModule):
    pass
```
here?

Reply:
Makes sense to me as the class is simply for tf logic not necessarily for framework setting. 

Will change it also for Torch.

Reply:
Short note: The `framework = "torch"` etc. was already in the `TorchRLModule`, so nothing to do but remvoing it from the BC Modules.